commcare_connect/program/tests/test_helpers.py

```diff
@@ -137,3 +141,122 @@ def test_failed_assessments(self):
         assert annotated_opp.workers_passing_assessment == 1
         assert annotated_opp.workers_starting_delivery == 2
         assert annotated_opp.percentage_conversion == 100.0
+
+
+@pytest.mark.django_db
+class TestDeliveryPerformanceReport:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.program_manager_org = OrganizationFactory.create()
```

Comment:
This would also require the `program_manager` attribute set to True to be a Program Manager Organization.
```suggestion
        self.program_manager_org = OrganizationFactory(program_manager=True)
```